application/src/main/java/org/opentripplanner/apis/gtfs/datafetchers/QueryTypeImpl.java

```diff
@@ -928,6 +928,34 @@ public DataFetcher<Iterable<VehicleRentalStation>> vehicleRentalStations() {
     };
   }
 
+  @Override
+  public DataFetcher<Iterable<Object>> vehicleRentalsByBbox() {
```

Comment:
Why not return `VehicleRentalPlace` here. You probably need to change the codegen configuration: https://github.com/opentripplanner/OpenTripPlanner/blob/f4bfedfc41d3b92df45927a72edcab28624510d6/application/src/main/java/org/opentripplanner/apis/gtfs/generated/graphql-codegen.yml

Reply:
I thought about it but couldn't get it right, I didn't know about the codegen configuration